pkg/version/version.go

```diff
@@ -19,7 +19,7 @@ import "fmt"
 // The below variables will be overrriden using ldflags set by goreleaser during the build process
 var (
 	// Version is the version string
-	Version = "v0.4.3"
+	Version = "v0.5.0-dev"
```

Comment:
We should put release tag

Reply:
Release tag will come later, we actually have broken tests now we need to fix before we release; setting this will just setup a pending release.

https://app.travis-ci.com/github/kevdowney/addon-manager/builds/246519616